duet/sys/config-user examples/sensorless/homex.g

```diff
@@ -6,7 +6,7 @@ M201 X1000 Y1000                ; reduce acceleration on X/Y to stop false trigg
 M915 P0:1 S3 R0 F0              ; both motors because corexy; Sensitivity 4, don’t take action, don’t filter
 
 G91                   	 	; use relative positioning
-G1 S1 X-330 F4000    		; move to home position
```

Comment:
Regression here, should be 330?

Reply:
OK, I don't know what regression means in this context :)

This value just needs to be more than X.
Just as a sanity check, I used the RRF configurator and it adds 5 to whatever your defined maximum X or Y is.
Since 310 is possible, we could be 315, but it doesn't hurt to have more does it?